Update old platform supports in IaC security documentation (#36504)
DataDog/documentation: content/en/security/code_security/iac_security/_index.md

    text: Detect and block exposed credentials with Datadog Secret Scanning
---

> Datadog Infrastructure as Code (IaC) Security detects misconfigurations in IaC configurations before they're deployed. It flags issues such as missing encryption or overly permissive access in files stored in your connected GitHub, GitLab, or Azure DevOps repositories. More informations on the support can be found on the [IaC Security Rules page][13].

Reviewer: ```suggestion
Datadog Infrastructure as Code (IaC) Security detects misconfigurations in IaC configurations before they're deployed. It flags issues such as missing encryption or overly permissive access in files stored in your connected GitHub, GitLab, or Azure DevOps repositories. For more information, see [IaC Security Rules][13].
```
